src/test/Testcafe/page-objects/common.js

```diff
@@ -0,0 +1,33 @@
+import { Selector, t } from 'testcafe';
+
+export default class commonPage { 
+    constructor () {
+        this.repoLogoutButton = Selector("#navbar > div.right.stackable.menu > div.ui.dropdown.jump.item.poping.up.active.visible > div > a:nth-child(10)");
+        this.designerLogoutButton = Selector("#navbarSupportedContent > ul:nth-child(2) > li > div > a:nth-child(4)");
```

Comment:
Is there no easier way to get the element? If we change the name of a class, you would have to go through all the tests that tests that involve that element?

Reply:
Fixed selector to find specific text in .ui.dropdown class, to make the selector less brittle